app/Http/Controllers/HomeController.php

```diff
@@ -23,6 +24,28 @@ public function __construct()
      */
     public function index()
     {
-        return view('home');
+        $user = auth()->user();
+        $following = $user->following()->get();
+        $followers = $user->followers()->get();
+        return view('home', compact('user', 'following', 'followers'));
```

Comment:
Please leave a space after each return
```suggestion
        $following = $user->following()->get();
        $followers = $user->followers()->get();
        
        return view('home', compact('user', 'following', 'followers'));
```